misc/theme-advancedopn/Makefile

```diff
@@ -0,0 +1,7 @@
+PLUGIN_NAME=		theme-advancedopn
```

Comment:
Just one quick thought here which is up to you but I'm weighing the benefit of theme-advancedopn vs. theme-advanced still derived from advancedtomato but obviously without new suffix which fits the current theme selection slightly better but I have no strong feelings either way. Thanks for your work!

Reply:
Good point. Honestly, I also have no strong feelings either way. So if you think advanced sounds better it's fine with me. I just felt "advanced" only may sound entitled but so does "advancedopn", so... 

Could also just be "modern" or something. Anything is fine with me =)